ext/bz2/tests/bug81092.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+Bug #81092 (fflush before stream_filter_remove corrupts stream)
+--SKIPIF--
+<?php
+if (!extension_loaded('bz2')) die('skip bz2 extension not available');
+?>
```

Comment:
Maybe use the new ``--EXTENSIONS--` section instead?

Reply:
This commit is for PHP-7.4 (regression has been introduced in PHP 7.4.17), so I can't use that yet. Shall do that when merging up, though.

Reply:
Good point forgot about that this is only on master